src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -62,7 +64,27 @@ const CharacterForm = ({ onClose }) => {
     return params[0] === formModeEnum.edit
   }, [params])
 
-  const character = useSelector(characterSelectors.character(params[1]))
+  const characterCacheId = client.cache.identify({
+    __typename: "Person",
+    id: params[1],
+  })
+
```

Comment:
This should not be getting called here, move it to the function that makes use of it